### ISSUE TITLE
[need help]Can I customize the button

### ISSUE THREAD

**Issue by qianshengta** [AppIntro/AppIntro#579]
**AppIntro Version**:
AppIntro Version：4.2.3

**Device/Android Version**:
null

**Issue details / Repro steps / Use case background**: 
My designers have strict requirements for interfaces，So can I customize the buttons on the interface？such as next and done botton, etc。In addition, my designer does not like the shadow effect when the button is clicked and the shadow effect when sliding to the edge. Could you provide a way to remove it?

**Your Code**: 
```
        SlideFragment firstFragment = SlideFragment.newInstance(1);
        SlideFragment secondFragment = SlideFragment.newInstance(2);
        SlideFragment thirdFragment = SlideFragment.newInstance(3);
        SlideFragment fourthFragment = SlideFragment.newInstance(4);
        addSlide(firstFragment);
        addSlide(secondFragment);
        addSlide(thirdFragment);
        addSlide(fourthFragment);
        showStatusBar(true);
        setProgressButtonEnabled(true);
        showSkipButton(false);
        setNavBarColor(R.color.colorBlue);
        setVibrate(true);
        setVibrateIntensity(30);
```

**Stack trace / LogCat**:
null

**Comment by Anutha02** [comment]
Partially fixed by #660

**Comment by Anutha02** [comment]
@cortinico Please close this
